explorer/tests/explorer_tests.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(test)]
+mod tests {
+
+    use chain_core::property::FromStr;
+
+    use jormungandr_automation::jormungandr::{
+        explorer::configuration::ExplorerParams, ConfigurationBuilder, Explorer,
+    };
+
+    use jormungandr_integration_tests::startup;
+    use jormungandr_lib::crypto::hash::Hash;
+    use jormungandr_lib::interfaces::ActiveSlotCoefficient;
+
+    pub fn get_invalid_block(explorer: &Explorer) {
+        let hash =
+            Hash::from_str("000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f")
+                .unwrap();
+
+        if let Some(errors) = explorer.block(hash).unwrap().errors {
+            for error in &errors {
```

Comment:
This doesn't panic if `errors` is empty, is that expected? I wonder if we could just compare `errors` to another `Vec` to make sure they're exactly the same